mmv1/provider/terraform_object_library.rb

```diff
@@ -66,7 +66,9 @@ def compile_common_files(output_folder, products, _common_compile_file)
                           ['google/provider_handwritten_endpoint.go',
                            'third_party/terraform/utils/provider_handwritten_endpoint.go.erb'],
                           ['google/transport.go',
-                           'third_party/terraform/utils/transport.go.erb']
+                           'third_party/terraform/utils/transport.go.erb'],
+                           ['google/mappers.go',
```

Comment:
the alignment for this line & the next are causing a rake test failure:

```
provider/terraform_object_library.rb:70:28: C: Layout/ArrayAlignment: Align the elements of an array literal if they span more than one line.
                           ['google/mappers.go', ...
                           ^^^^^^^^^^^^^^^^^^^^^
provider/terraform_object_library.rb:71:30: C: Layout/ArrayAlignment: Align the elements of an array literal if they span more than one line.
                             'templates/validator/mappers/mappers.go.erb']
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

It should resolve if you align this the same way the previous lines are aligned.